src/components/Table.js

```diff
@@ -178,7 +184,7 @@ export default class Table extends React.Component<TableDefaultProps, TableProps
         selectedRowIds = [rowId];
       }
       this.props.onSelect(selectedRowIds, rowId);
-    } else {
+    } else if (!this.props.noEmptySelection) {
```

Comment:
We only want to disable deselection if the current selection has only one item in it, right? The way it is now, it looks like you won't be able to ever deselect anything in a multi-select table...